Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs

```diff
@@ -0,0 +1,593 @@
+
+            public long threadId;
+        }
+
+        public sealed class ProcessLoadEvent : Event {
+            public const string Name = "legacyProcessLoad";
+
+            public override string name => Name;
+
+            public long threadId;
+        }
+
+        public sealed class BreakpointSetEvent : Event {
+            public const string Name = "legacyBreakpointSet";
+
+            public override string name => Name;
+
+            public int breakpointId;
+        }
+
+        public sealed class BreakpointFailedEvent : Event {
+            public const string Name = "legacyBreakpointFailed";
+
+            public override string name => Name;
+
+            public int breakpointId;
+        }
+
+        public sealed class DebuggerOutputEvent : Event {
+            public const string Name = "legacyDebuggerOutput";
+
+            public override string name => Name;
+
+            public long threadId;
+            public string output;
+        }
+
+        public sealed class ExecutionResultEvent : Event {
+            public const string Name = "legacyExecutionResult";
+
+            public override string name => Name;
+
+            public int executionId;
+            public PythonObject obj;
+        }
+
+        public sealed class ExecutionExceptionEvent : Event {
+            public const string Name = "legacyExecutionException";
+
+            public override string name => Name;
+
+            public int executionId;
+            public string exceptionText;
+        }
+
+        public sealed class EnumChildrenEvent : Event {
+            public const string Name = "legacyEnumChildrenResult";
+
+            public override string name => Name;
+
+            public int executionId;
+            public EnumChildrenItem[] children;
+        }
+
+        public sealed class ThreadFrameListEvent : Event {
+            public const string Name = "legacyThreadFrameList";
+
+            public override string name => Name;
+
+            public long threadId;
+            public string threadName;
+            public ThreadFrameItem[] threadFrames;
+        }
+
+        public sealed class InternalErrorEvent : Event {
+            public const string Name = "legacyInternalError";
+
+            public override string name => Name;
+
+            public string output;
+        }
+
+        //////////////////////////////////////////////////////////////////////
+        /// Protocol types
+        //////////////////////////////////////////////////////////////////////
+
+        [Flags]
+        public enum EvaluationResultFlags {
+            None = 0,
+            Expandable = 1,
+            MethodCall = 2,
+            SideEffects = 4,
+            Raw = 8,
+            HasRawRepr = 16,
+        }
+
+        public enum LanguageKind {
+            Python = 0,
+            Django = 1,
+        }
+
+        public enum FrameKind {
+            None,
+            Python,
+            Django,
```

Comment:
Let's start with explicit values on all enums, just to make it clear that they shouldn't be reordered.